src/lib/tracer.ts

```diff
@@ -26,9 +26,9 @@ export function init() {
 const createCommand = (command: string) => <T>(
   promise: Promise<T>
 ): Promise<T> => {
-  const parentScope = tracer.scopeManager().active()
+  const parentScope = tracer.scope().active()
   const span = tracer.startSpan("memcached", {
-    childOf: parentScope && parentScope.span(),
+    childOf: parentScope || undefined,
```

Comment:
I had to do this for type reasons, but maybe there's a nicer, more idiomatic way?